apps/condo/domains/ticket/schema/TicketLastTimeViewedService.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Generated by `createservice ticket.TicketLastTimeViewedService`
+ */
+
+const { GQLCustomSchema } = require('@core/keystone/schema')
+const access = require('@condo/domains/ticket/access/TicketLastTimeViewedService')
+const REDIS_URL = require('@core/config').REDIS_URL
+const dataSource = new (require('ioredis'))(REDIS_URL)
+
+const TicketLastTimeViewedService = new GQLCustomSchema('TicketLastTimeViewedService', {
+    types: [
+        {
+            access: true,
+            type: 'input SetTicketLastTimeViewedInput { viewedAt: Float! }',
+        },
+        {
+            access: true,
+            type: 'type SetTicketLastTimeViewedOutput { status: String! }',
+        },
+        {
+            access: true,
+            type: 'type GetTicketLastTimeViewedOutput { syncedTicketLastTimeViewed: Float! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canTicketLastTimeViewed,
+            schema: 'setTicketLastTimeViewed(data: SetTicketLastTimeViewedInput!): SetTicketLastTimeViewedOutput',
+            resolver: async (parent, args, context, info, extra = {}) => {
+                const { data: { viewedAt } } = args
+                dataSource.set(context.authedItem.id, viewedAt)
```

Comment:
This solution does not allows to implement another counting feature for another records.
Maybe, to use some scoped key, like `lastTimeTicketsViewedByUser-[id]`.